src/Jaeger.Core/Senders/README.md

```diff
@@ -0,0 +1,62 @@
+# Senders
+Use senders to define what happens when a span gets reported to a `RemoteReporter`.
+
+## ISender
+Defines the methods called by the `RemoteReporter`. Each instance of the `ISender` therefore handles the transport of the spans reported to the target system.
+
+It is usually instantiated through `ISenderFactory.GetSender()` which receives an instance of `Configuration.SenderConfiguration` for configuring the sender.
+
+## ISenderFactory
+It has a fixed and unique `FactoryName` with that it's registered to the `SenderResolver`. If multiple `ISenderFactory` instances are registered to one `SenderResolver`, the configuration `JAEGER_SENDER_FACTORY` needs to be set to the `FactoryName` to select the targeted instance.
+
+## SenderResolver
+It resolves an `ISender` instance based on `Configuration.SenderConfiguration`. `Configuration.SenderConfiguration.SenderFactory` (usually coming from `JAEGER_SENDER_FACTORY`) is used to select an `ISenderFactory` by it's matching `FactoryName`. This is only needed if multiple `ISenderFactory` instances are registered to this `SenderResolver`.
+
+If the factory to use is not found or was ambiguous due to no `SenderFactory` being defined, `NoopSender.Instance` is returned instead.
+
+If a factory was found, `ISenderFactory.GetSender()` will be used to get a configured instance of `ISender` to be returned to the caller of `SenderResolver.Resolve()`.
+
+```C#
+var senderResolver = new SenderResolver(loggerFactory)
+	.RegisterSenderFactory<ThriftSenderFactory>();
+var senderConfiguration = new Configuration.SenderConfiguration(loggerFactory)
+	.WithSenderResolver(senderResolver)	// optional, defaults to Configuration.SenderConfiguration.DefaultSenderResolver
+	.WithSenderFactory("thrift");		// optional if only one Factory registered to senderResolver
```

Comment:
```suggestion
	.WithSenderFactory("thrift");		// optional when only one Factory registered to senderResolver
```

Reply:
Thanks, updated. And I also replaced the constant string with `ThriftSenderFactory.Name` and added the note that `"thrift"` is to be used if `JAEGER_SENDER_FACTORY` is necessary.